atomics/include/desul/atomics/Fetch_Op_GCC.hpp

```diff
@@ -55,6 +55,20 @@ DESUL_IMPL_GCC_HOST_ATOMIC_FETCH_OP(_xor, std::is_integral)
 DESUL_IMPL_GCC_HOST_ATOMIC_FETCH_OP(_or, std::is_integral)
 DESUL_IMPL_GCC_HOST_ATOMIC_FETCH_OP(_nand, std::is_integral)
 
+#if defined(__clang__)
+#if (__clang_major__ >= 17) && \
+    (!defined(__INTEL_LLVM_COMPILER) || __INTEL_LLVM_COMPILER >= 20240000)
```

Comment:
https://godbolt.org/z/qKaWhWdqW

Reply:
(We had not noticed on the Kokkos side because we bumped our minimum requirement to 2024.2.1)